model/StringArray.go

```diff
@@ -0,0 +1,32 @@
+package model
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+
+	_ "github.com/go-sql-driver/mysql" // Load MySQL Driver
+)
+
+type StringArray []string
+
+func (sa StringArray) Value() (driver.Value, error) {
+	b, err := json.Marshal(sa)
+	if err != nil {
+		return nil, err
+	}
+	return string(b), nil
```

Comment:
Since we are getting a byte slice, it's not needed to convert to string. We can just directly do `return json.Marshal(j)`